Fix dlls are installed depending on wine type.
See: https://github.com/wineasio/wineasio/pull/77
See: https://github.com/Winetricks/winetricks/blob/88643d0577a1fc4ea484e84f2f5a141686f146eb/src/winetricks#L5052-L5181

diff --git a/wineasio-register b/wineasio-register
index 862fb0e..c7fbe73 100755
--- a/wineasio-register
+++ b/wineasio-register
@@ -36,8 +36,15 @@ status32="not found"
 for u in ${u32[@]}; do
     w=$(echo ${u} | sed -e 's|/i386-unix/wineasio32.dll.so|/i386-windows/wineasio32.dll|g')
     if [ -e "${u}" ] && [ -e "${w}" ]; then
-        cp -v "${w}" "${WINEPREFIX}/drive_c/windows/system32"
-        regsvr32 "${u}"
+        if command -v wine64 > /dev/null && [ -d "${WINEPREFIX}/drive_c/windows/syswow64" ]; then
+            cp -v "${w}" "${WINEPREFIX}/drive_c/windows/syswow64"
+            W_SYSTEM32_DLLS_WIN="C:\\windows\\syswow64"
+        else
+            cp -v "${w}" "${WINEPREFIX}/drive_c/windows/system32"
+            W_SYSTEM32_DLLS_WIN="C:\\windows\\system32"
+        fi
+        wine "${W_SYSTEM32_DLLS_WIN}\\regsvr32.exe" "${u}"
+        
         code=$?
         if [ $code -eq 0 ]; then
             status32="registered"
